mapbox-gl-directions.d.ts

```diff
@@ -0,0 +1,38 @@
+declare module MapBoxDirections {
+    export let accessToken: string;
+    
+    export type WayPoint = [number, number];
```

Comment:
These can app point one thing `Point`, I guess the name helps for meaning, but the type system will resolve them the same.